config/locales/ar.yml

```diff
@@ -224,4 +224,4 @@ ar:
   ZM: زامبيا
   ZW: زمبابوي
 
-  missing: <em>[Translation N/A]</em>
+  missing: ""
```

Comment:
Discussed with Motasem and decided to not show anything when translation is missing.